src/test/groovy/org/osiam/test/integration/RegistrationIT.groovy

```diff
@@ -307,6 +331,16 @@ class RegistrationIT extends AbstractIT {
         registeredExtension2.getField('age', ExtensionFieldType.STRING) != null
         registeredExtension2.getField('age', ExtensionFieldType.STRING) == '12'
         registeredUser.getGroups().get(0).getDisplay().equalsIgnoreCase("Test")
+
+        Thread.sleep(5000)
```

Comment:
I know this is a bad idea to use `Thread.sleep` in a test... should I use [CountDownLatch](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/CountDownLatch.html) instead? Wait until one email was received by greenmail, right?